src/pages/podcasts.js

```diff
@@ -1,11 +1,26 @@
+import PageLayout from '../components/PageLayout';
 import React from 'react';
-import cx from 'classnames';
 import SEO from '../components/Seo';
-import PageLayout from '../components/PageLayout';
+import cx from 'classnames';
 import podcastsHeader from '../images/podcasts/podcasts.jpg';
 import styles from './podcasts.module.scss';
 
 const PodcastsPage = () => {
+  const podcastIds = ['1225223', '1677727', '1677670'];
+
+  const podcastEmbeds = podcastIds.map((podcastId) => {
```

Comment:
[nit] I generally prefer to have all the JSX inline so that you don't have to bounce back and forth when understanding this part of the UI.

This change is GREAT by the way. I really like that this makes it easy to add new podcast sections in the future.